src/components/Bar.jsx

```diff
@@ -0,0 +1,90 @@
+/* global __TARGET__ */
+/* global __PIWIK_TRACKER_URL__  __PIWIK_SITEID__ __PIWIK_DIMENSION_ID_APP__ */
+
+import React, { Component } from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+import { shouldEnableTracking, getTracker, configureTracker } from '../lib/piwik'
+
+// import Apps from './Apps'
+// import Settings from './Settings'
+// import Drawer from './Drawer'
+// import Claudy from './Claudy'
+
+class Bar extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      claudyConfig: null, // no claudy by default
+      claudyOpened: false,
+      drawerVisible: false,
+      usageTracker: null
+    }
+  }
+
+  componentDidMount () {
+    // if tracking enabled, init the piwik tracker
+    if (shouldEnableTracking()) {
+      const trackerInstance = getTracker(__PIWIK_TRACKER_URL__, __PIWIK_SITEID__, false, false)
+      configureTracker({
+        appDimensionId: __PIWIK_DIMENSION_ID_APP__,
+        app: 'Cozy Bar',
+        heartbeat: 0
+      })
+      trackerInstance.push(['disableHeartBeatTimer']) // undocumented, see https://github.com/piwik/piwik/blob/3.x-dev/js/piwik.js#L6544
+      this.set({usageTracker: trackerInstance})
+    }
+  }
+
+  toggleDrawer () {
+    // don't allow to toggle the drawer if claudy opened
+    if (this.state.claudyOpened) return
+    this.setState({ drawerVisible: !this.state.drawerVisible })
```

Comment:
Good point, thanks for this ressource 👍 but I think it's too far for now in this PR.